style.css

```diff
@@ -1,3 +1,230 @@
+/* GENERIC STYLING */ 
+
 * {
 	box-sizing: border-box;
+	font-family: Helvetica;
+	font-size: 10px;
+}
+
+h1 {
+	font-family: cursive;
+	color: #ebedf0; /*a grey-ish white*/
+	font-size: 7rem;
+	margin: 20px auto;
+}
+
+h2 {
+	font-size: 3rem ;
+	color: #d65a64; /* cute pink colour*/	
+	margin: 0 50px;
+}
+
+h3 {
+	color: #ebedf0;	
+	font-size: 2rem;	
+}
+
+p {
+	color: #ebedf0;
+	font-size: 1.8rem;
+}
+
+a {
+	text-decoration: none;
+	font-weight: bold;
+	color: #d65a64; 
+	font-size: 1.8rem;
+}
+
+.aboutMe_section, .portfolio_section, .contact_section {
+	padding-top: 30px;
+	padding-bottom: 30px;
+}
+
+/* HEADER SECTION*/
+
+header {
+	display: flex;
+	flex-direction: column;
+	background-color: #262323;
+	padding: 20px;	
+}
+
+nav {
+	display: flex;
+	margin: 20px auto;
+	gap: 20px;
+	justify-content: space-evenly;
+}
+
+nav a {
+	font-size: 2.5rem;
+}
+
+
+/* INTRODUCTION/WELCOME SECTION */
+
+.introduction_section {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-end;
+	justify-content: center;
+	column-gap: 20px;
+	width: 100%;
+	height: 900px;
+	background-image: url(images/horse_head_nebula.jpg); /* a place-holder image */ 
+	background-size:cover; /* allows to get the background in the whole section*/
+}
+
+.introduction_section h2 {
+	color: white;
+	font-size: 5rem;
+	font-variant: small-caps;
+	margin-left: 30%;
+	margin-right: 15%;
+}
+
+.introduction_section p {
+	background-color: rgba(0,0,0,0.5);
+	color: white;
+	margin-left: 30%;
+	margin-right: 15%;	
+	font-size: 3.5rem;
+	font-variant: small-caps;	
+}
+
+
+/* ABOUT ME SECTION */
+
+.aboutMe_section {
+	display: flex;
+  flex-direction: column;
+	align-items: flex-start;
+	justify-content: center;
+	background-color: #262323;	
+}
+
+.aboutMe_card {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: flex-start;
+	align-items: center;
+	margin-left: 50px;
+	gap: 75px;
+}
+
+.aboutMe_section img {
+	width: 200px;
+	height: 250px;
+	border-radius: 10px;
+}
+
+.aboutMe_text {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start ;		
+	max-width: 1000px;
+}
+
+.aboutMe_text p, .aboutMe_text a {
+	font-size: 2.4rem;
+}
+
+.aboutMe_text a {
+	color: white;
+	background-color: #d65a64;
+	padding: 5px;
+	border-radius: 10px;
+}
+
+/* PORTFOLIO SECTION */ 
+
+.portfolio_section {
+	background-color: #171616;
+}
+
+.portfolio_section h2 {
+	margin: 0 50px;
+}
+
+.project_cards_container {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: flex-start;
+	margin: 20px 50px;
+	gap: 50px 200px;
+}
+
+.project_card {
+	border: 2px solid #262323;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-starts;
+	max-width: 400px;
+	padding: 20px;
+	border-radius: 10px;
+	column-gap: 50px;
+
+}
+
+.project_card img, .project_card p {
+	height: 200px;
```

Comment:
the p tag should not have a set height, because it can cause this problem:
![image](https://user-images.githubusercontent.com/25844355/213475879-c4b1d94d-5bad-48fb-9c4b-e9b6785b2de9.png)

Reply:
OK, I removed the p selector.